cmd/format/format.go

```diff
@@ -204,7 +204,7 @@ func Run(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, paths []string)
 
 	if formatErr != nil {
 		// return an error if any formatting failures were detected
-		return formatErr
+		return formatErr //nolint:wrapcheck
```

Comment:
Left out because I was unsure if this should append something to the error message.